### DISCUSSION TITLE
Run a custom sandboxie build without test mode

### DISCUSSION THREAD

**Discussion by merlijnzeijden** [sandboxie-plus/Sandboxie#3085] [Q&A Feedback]
I'm wondering if it is possible to run a custom Sandboxie build without having windows in test mode. 
As far as I understand the main issue is the certificate on the driver. Lets say in my custom build I don't make any changes to the driver, could I just use the official signed driver for my build?

**Comment by isaak654** [comment]
See https://github.com/sandboxie-plus/Sandboxie/issues/1388#issuecomment-1153802583 for a list of options.

**Comment by DavidXanatos** [comment]
This depends in the end on which functionality you want to change, official signed driver checks for the integrity of the system service (sbiesvc) and the user agents (sandman/sbiectrl) these must be signed as well.
What must not be signed is the sbiedll.dll so if you only want to change that port you can.
